bin/write-mime-multipart

```diff
@@ -9,7 +9,7 @@ from email import encoders
 from email.mime.base import MIMEBase
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
-from optparse import OptionParser
+from optparse import OptionParser  # pylint: disable=W0402
```

Comment:
This import is used; do we know why pylint is failing to detect that?

Reply:
That's a deprecation warning: pylint wants us to switch to argparse. It's indeed the right thing to do, but I was trying to avoid introducing non-trivial changes together with the CI, with this in mind:
1. Make CI pass with what we have (modulo minor fixes)
1. Be strict on new code
1. Gradually improve existing code as we touch it